src/libraries/BotBuilder/Microsoft.Agents.BotBuilder/UserAuth/TokenService/AzureBotUserAuthorization.cs

```diff
@@ -12,21 +14,31 @@ namespace Microsoft.Agents.BotBuilder.UserAuth.TokenService
     /// <summary>
     /// Handles user authentication using The Azure Bot Token Service.
     /// </summary>
-    public class OAuthAuthentication : IUserAuthentication
+    public class AzureBotUserAuthorization : IUserAuthorization
```

Comment:
does this need to be public?

Reply:
Well... the intent is that there could be other providers.  For examples, TeamsSsoAut